samples/serialize_arguments/DurableOrchestration/__init__.py

```diff
@@ -1,28 +1,28 @@
-import logging
-
-import azure.functions as func
-import azure.durable_functions as df
-from ..shared_code.MyClasses import SerializableClass # TODO: this import is highlight 'red' in VSCode, but works at runtime
-
-def orchestrator_function(context: df.DurableOrchestrationContext):
-    """This function provides the core function chaining orchestration logic
-
-    Parameters
-    ----------
-    context: DurableOrchestrationContext
-        This context has the past history and the durable orchestration API's to
-        create orchestrations
-
-    Returns
-    -------
-    int
-        The number contained in the SerializableClass input object
-    """
-    input_: SerializableClass = context.get_input()
-    num1: int = input_.show_number()
-
-    # The custom class is also correctly serialized when calling an activity
-    num2 = yield context.call_activity("DurableActivity", SerializableClass(5))
-    return num1
-
-main = df.Orchestrator.create(orchestrator_function)
+import logging
+
+import azure.functions as func
+import azure.durable_functions as df
+from ..shared_code.MyClasses import SerializableClass # TODO: this import is highlight 'red' in VSCode, but works at runtime
```

Comment:
Did we ever figure out why this import is highlighted red in VSCode?

Reply:
It is no longer highlighted as red for me, so I'm removing this comment